### PR TITLE
W-14501939-rpa-auditlogs-se

### PR DESCRIPTION
**Pull request by sebermuley**
New RPA paragraph
with audit logging tables

# Writer's Quality Checklist

Before merging your PR, did you:

- [ ] Run spell checker
- [ ] Run link checker to check for broken xrefs
- [ ] Check for orphan files
- [ ] Perform a local build and do a final visual check of your content, including checking for:
  - Broken images
  - Dead links
  - Correct rendering of partials if they are used in your content
  - Formatting issues, such as:
    - Misnumbered ordered lists (steps) or incorrectly nested unordered lists
    - Messed up tables
    - Proper indentation
    - Correct header levels
- [ ] Receive final review and signoff from:
  - Technical SME
  - Product Manager
  - Editor or peer reviewer
  - Reporter, if this content is in response to a reported issue (internal or external feedback)
- [ ] If applicable, verify that the software actually got released

### CODE REVIEW

**Comment by sebermuley** [comment]
SME review in progress

File: modules/ROOT/pages/audit-logging.adoc [hunk 1]
```diff
@@ -2024,6 +2024,539 @@ Properties: Review object
...
+* Execution taskbar option 
+* Schedules
+** Schedule name
+* Bots
+** Bot name
+** Bot ID
+** Assigned sessions
+
+
+| Delete run Configuration
```
Comment (kevintroller) [inline modules/ROOT/pages/audit-logging.adoc]:
```suggestion
| Delete run configuration
```

File: modules/ROOT/pages/audit-logging.adoc [hunk 2]
```diff
@@ -2024,6 +2024,539 @@ Properties: Review object
...
+|  Subaction: None +
+Properties:
+
+* Process name
+* Process ID
+* Process phase
+* Run configuration name
+
+
+| Deploy run Configuration
```
Comment (kevintroller) [inline modules/ROOT/pages/audit-logging.adoc]:
```suggestion
| Deploy run configuration
```

Comment (kevintroller) [reply modules/ROOT/pages/audit-logging.adoc]: DITTO with all parameters names

File: modules/ROOT/pages/audit-logging.adoc [hunk 3]
```diff
@@ -2024,6 +2024,539 @@ Properties: Review object
...
+| Bot
+| Bot ID
+| N/A
+| Register 
+| Subaction: none + 
+
+Properties:
+
+* Bot name
+* Api key name
```
Comment (kevintroller) [inline modules/ROOT/pages/audit-logging.adoc]:
```suggestion
* API Key name
```

File: modules/ROOT/pages/audit-logging.adoc [hunk 4]
```diff
@@ -2024,6 +2024,539 @@ Properties: Review object
...
```
Comment (kevintroller) [inline modules/ROOT/pages/audit-logging.adoc]: DITTO

File: modules/ROOT/pages/audit-logging.adoc [hunk 5]
```diff
@@ -2024,6 +2024,539 @@ Properties: Review object
...
+|===
+|User Action
+|Object Type
+|Object
+|Parent
+|Action
+|Payload
+
+
+| Create Application
```
Comment (kevintroller) [inline modules/ROOT/pages/audit-logging.adoc:2455]:
```suggestion
| Create application
```

Comment (kevintroller) [reply modules/ROOT/pages/audit-logging.adoc:2455]: Not sure if these are UI elements. If they are capitalized then use the same name.

File: modules/ROOT/pages/audit-logging.adoc [hunk 6]
```diff
@@ -2024,6 +2024,539 @@ Properties: Review object
...
+
+Properties:
+
+* Application name
+* Application time zone
+* Downtimes
+** Downtime name
+
+
+| Edit Application
```
Comment (kevintroller) [inline modules/ROOT/pages/audit-logging.adoc:2470]:
```suggestion
| Edit application
```

**Review by kevintroller — DISMISSED** [review]
Approved. Added minor suggestions. Ignore them if they are UI elements.

File: modules/ROOT/pages/audit-logging.adoc (hunk 5 above)
Comment (sebermuley) [reply modules/ROOT/pages/audit-logging.adoc:2455]: This is kind of a UI element, so I left it upper case (It's not an application but a kind of tag for which applications the bot can access)

File: modules/ROOT/pages/audit-logging.adoc (hunk 6 above)
Comment (sebermuley) [reply modules/ROOT/pages/audit-logging.adoc:2470]: Ditto

**Review by kevintroller — APPROVED** [review]

**Pull request merged**